docs/tutorial-adding-a-new-command-to-the-vs-code-extension.md

```diff
@@ -0,0 +1,69 @@
+# Tutorial Adding a New Command to the VS Code Extension
```

Comment:
Feel free to add a colon or other formatting here. The title will show up wherever this page is referenced.

```suggestion
# Tutorial: Adding a New Command to the VS Code Extension
```